test/extended/node_sizing.go

```diff
@@ -0,0 +1,529 @@
+package extended
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+	"k8s.io/utils/ptr"
+
+	mcfgv1 "github.com/openshift/api/machineconfiguration/v1"
+	machineconfigclient "github.com/openshift/client-go/machineconfiguration/clientset/versioned"
+	"github.com/openshift/machine-config-operator/pkg/controller/common"
+	exutil "github.com/openshift/machine-config-operator/test/extended-priv/util"
+)
+
+const (
+	nodeSizingTestMCPName       = "node-sizing-test"
+	nodeSizingTestKubeletName   = "auto-sizing-enabled"
+	nodeSizingTestPodName       = "node-sizing-test"
+	nodeSizingAutoSizingPodName = "node-sizing-autosizing-test"
+	nodeSizingEnvFile           = "/etc/node-sizing-enabled.env"
+	nodeSizingEnvHostFile       = "/host/etc/node-sizing-enabled.env"
+	nodeSizingEnvKey            = "NODE_SIZING_ENABLED"
+	mcpPoolLabel                = "machineconfiguration.openshift.io/pool"
+)
+
+var _ = g.Describe("[sig-mco][Suite:openshift/machine-config-operator/disruptive][Serial] Node sizing", func() {
```

Comment:
Thank you for the fixups. The MCO team has just had a discussion on this, and we believe that these tests should live in a new test suite. Adding additional non-MCO managed tests increases the chances of inter-test conflicts, and also limits our already-stretched test timeout capacity. In this case, we suggest:
1. create a new suite (perhaps openshift/node/disruptive or similar)
2. create the corresponding jobs to exercise the suite (it doesn't look like we're using this for feature promotion, just automated testing, so perhaps 1-2 jobs is fine, vs the current MCO FG tests which run in 14 jobs)
3. move this back to the origin repo and attach sig-node to the list